doc/source/missing_data.rst

```diff
@@ -36,7 +36,7 @@ When / why does data become missing?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Some might quibble over our usage of *missing*. By "missing" we simply mean
-**null** or "not present for whatever reason". Many data sets simply arrive with
+**na** or "not present for whatever reason". Many data sets simply arrive with
```

Comment:
I would personally use 'NA' instead of 'na' in prose. Thoughts? 
Would also add this stands for 'Not Available' this first time